programs/ssl/ssl_client2.c

```diff
@@ -1949,7 +2006,20 @@ int main( int argc, char *argv[] )
         }
 #endif
 
-    mbedtls_ssl_conf_rng( &conf, mbedtls_ctr_drbg_random, &ctr_drbg );
+    if (opt.reproducible)
+    {
+        srand( 1 );
+        mbedtls_ssl_conf_rng( &conf, dummy_random, &ctr_drbg );
+#if defined(MBEDTLS_PLATFORM_TIME_ALT)
+        mbedtls_platform_set_time( dummy_constant_time );
+#else
+        fprintf( stderr, "Warning: reproduce without constant time\n" );
```

Comment:
Replace `reproduce` with `reproducible option used`

Reply:
Done.
Should we exit in this case ?

Reply:
I don't see the `option used` part, so the sentence still doesn't make much sense.

If the library is configured without time, shouldn't that be reproducible? It's only if MBEDTLS_HAVE_TIME is set that we should have to worry about setting up a dummy time, right?

Reply:
> I don't see the option used

Sorry, I read too quickly.
Added

> If the library is configured without time, shouldn't that be reproducible?

I think so since dummy random will be used instead of time (if I am correct).
So, I just pushed another commit to take this into account